cmd/kubeadm/app/util/error.go

```diff
@@ -26,9 +26,12 @@ import (
 )
 
 const (
+	// DefaultErrorExitCode defines default exit code for failed action
```

Comment:
remove default in all these comments: `...defines the exit code for a failed action generally`